pkg/image/source.go

```diff
@@ -35,6 +37,7 @@ var sourceStr = [...]string{
 	"OciDirectory",
 	"OciTarball",
 	"OciRegistry",
+	"PodmanDaemon",
```

Comment:
This is what I was referencing in the above comment about sharing the value for when we parse the type of Daemon:
`internal/docker/client.go`